mappings/net/minecraft/server/SaveLoading.mapping

```diff
@@ -0,0 +1,35 @@
+CLASS net/minecraft/class_7237 net/minecraft/server/SaveLoading
+	METHOD method_42096 (Lnet/minecraft/class_6860;Lnet/minecraft/class_5350;Ljava/lang/Throwable;)V
+		ARG 1 dataPackContents
+		ARG 2 throwable
+	METHOD method_42097 (Lnet/minecraft/class_5455$class_6890;Lnet/minecraft/class_7237$class_7239;Lnet/minecraft/class_6860;Ljava/lang/Object;Lnet/minecraft/class_5350;)Ljava/lang/Object;
+		ARG 4 dataPackContents
+	METHOD method_42098 load (Lnet/minecraft/class_7237$class_6906;Lnet/minecraft/class_7237$class_6907;Lnet/minecraft/class_7237$class_7239;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
+		ARG 0 functionLoaderConfig
+		ARG 1 savePropertiesSupplier
+		ARG 2 saveUserFactory
+		ARG 3 prepareExecutor
+		ARG 4 applyExecutor
+	CLASS class_6906 FunctionLoaderConfig
```

Comment:
More like `ServerConfig`, as this record is mainly used for server stuff than just for function stuff.

Reply:
I think this name was from 1.18.2 when this was actually used to configure FunctionLoader. We can rename it, yeah.